stdlib/dynarray.mli

```diff
@@ -308,6 +308,31 @@ val filter_map : ('a -> 'b option) -> 'a t -> 'b t
     ignoring strings that cannot be converted to integers.
 *)
 
+(** {1:comparison Comparison functions}
+
+    Comparison functions iterate over their two dynarray inputs; it is
```

Comment:
I found over `their two dynarray inputs` a bit hard to parse, the  `two` can be dropped, and perhaps even the `dynarray`, the context seems quite clear. I would go with the first of the following, but the others are fine too:

```
Comparison functions iterate over their arguments
Comparison functions iterate over their dynarray arguments
Comparison functions iterate over their inputs
Comparison functions iterate over their dynarray inputs
```